app/controllers/v0/sessions_controller.rb

```diff
@@ -27,7 +27,7 @@ def saml_callback
         render json: @session, status: :created
       else
         # TODO: also need to make sure error json conforms to api spec
-        render json: { errors: @saml_response.errors.full_messages }, status: :forbidden
+        render json: { errors: @saml_response.errors }, status: :forbidden
```

Comment:
If you raise a subclass of `Common::Exceptions::BaseError`, our application rescue_from will render it as JSON-API style JSON.